website/docs/getting-started/setup/instance-configuration/authentication/openid-connect-oidc/aws-cognito.md

```diff
@@ -0,0 +1,83 @@
+# AWS Cognito
+
+[AWS Cognito](https://aws.amazon.com/cognito/) lets you add user sign-up and authentication to your mobile and web apps. To configure OpenID Connect(OIDC) within Appsmith using AWS Cognito as an OIDC provider, follow these steps:
+
+:::info
+OpenID Connect is available only in the [**business edition**](https://www.appsmith.com/pricing) for **self-hosted instances,** and only the **Superuser** of your **Appsmith Instance** can set up OIDC.
+:::
+
+## Configure a user pool
+
+Log in to your AWS account, open AWS Cognito and follow the steps below:
+
+1. Click the "Create User pool" button to create a new user pool.
+2. **Step 1 - Configure sign-in experience**: select "Email" in the Cognito user pool sign-in options. Click *Next*
+
+    ![](/img/Step1-AWS_Cognito.png)
+
+3. **Step 2 - Configure security requirements:** select "No MFA" in Multi-factor authentication. Click *Next*. 
+
+    ![](/img/Step2-AWS_Cognito.png)
+
+4. **Step 3 - configure sign-up experience:** keep the default configurations and click *Next*.
+5. **Step 4 -Configure message delivery:** you can choose either of the options - *Send email with Amazon SES* or *Send email with Cognito*. Please pick *Send email with Cognito* for this setup.
+6.  **Step 5 - Integrate your app:**
+    1. Enter a user pool name.
+    2. Under *Initial app client*, add an app client name and select **Generate a client secret** under Client secret section.
+
+    ![](/img/AWS-cognito_App_client.png)
+
+7. **Step 6 - Review Configurations:** review all the selected configurations and proceed once done.
+8. Click the user pool you've created, go to the integrations tab and follow the steps below 
+    1. In the domains section, click on the actions dropdown and select **Create a Cognito domain**. You can also select **Create a custom domain** to add a domain that you own.
+
+    ![](/img/AWS-cognito_create-cognito-domain.png)
+
+    2. Enter a domain prefix in the **Create Cognito domain** screen and click on **Create Cognito domain** button.
+
+        ![](/img/AWS-cognito_create_domain.png)
+
+9. On the user pool home, scroll down to the **App client list** and open the App Client that you created.
+    1. Copy the client ID and client secret that will be added to the OIDC configurations in Appsmith.
+
+    ![](/img/AWS-cognito_client-creds.png)
+
+    2. Scroll down to the Hosted UI section and click Edit. Click the Add Callback URL button and enter Redirect copied from the OIDC window in Appsmith’s Admin Settings.
+
+## Configure AWS Cognito with Appsmith
+
+On your Appsmith instance, go to *Admin Settings> Authentication> OIDC*, click Enable and follow the steps below:
```

Comment:
this line and point no 1 should be above **Create user pool** section. You can name the section 

**Enable OIDC on Appsmith**